### PR TITLE
Chunked file upload

### PR DESCRIPTION
**Pull request by antonmolodykh**

### CODE REVIEW

**Comment by antonmolodykh** [comment]
@korvyashka hmm. I suppose, in case of uploading fails, we need to send a request to delete the file in the API. right?

**Comment by korvyashka** [comment]
@antonmolodykh Or some scheduled cleanup, as in the case of big uploads failed in the middle - we'll have a big unusable file in the end device.

**Comment by vpetersson** [comment]
@antonmolodykh did you address the cleanup comment?

**Comment by antonmolodykh** [comment]
@vpetersson not yet. Tomorrow everything will be done. I'll describe in details in the today report.

**Comment by vpetersson** [comment]
Ok, i subtract @korvyashka LGTM then ;)

File: ansible/roles/screenly/tasks/main.yml [hunk 1]
```diff
@@ -57,6 +57,12 @@
     owner: root
     group: root
 
+- cron:
+    name: Cleanup screenly_assets
+    special_time: reboot
```
Comment (vpetersson) [inline ansible/roles/screenly/tasks/main.yml]: Nodes can have weeks or months uptime, so let's run this at say 1AM every night instead.

**Pull request merged**